### PR TITLE
Update app and proxy for GCP integration

### PR DESCRIPTION
**Pull request by diazandr3s**
Signed-off-by: Andres <diazandr3s@gmail.com>

### CODE REVIEW

**Review by SachidanandAlle — APPROVED** [review]

**Comment by SachidanandAlle** [comment]
Some unit test fails.. please make sure you have tested non-gcp (basic orthanc) setup for basic workflow

**Comment by diazandr3s** [comment]
> Some unit test fails.. please make sure you have tested non-gcp (basic orthanc) setup for basic workflow

It's the `UnboundLocalError: local variable` ... it seems it didn't like the `dw_client` being within the conditional. Have you faced this before?

**Comment by SachidanandAlle** [comment]
possibly @tangy5 can help you to fix this issue

**Comment by tangy5** [comment]
Thanks. I see, the get() function in the mockHttpClient should be changed accordingly....

File: monailabel/endpoints/proxy.py [hunk 1]
```diff
@@ -80,6 +82,11 @@ async def proxy_dicom(op: str, path: str, response: Response):
     return response
 
 
+@router.get("/dicom/{path:path}", include_in_schema=False)
```
Comment (SachidanandAlle) [inline monailabel/endpoints/proxy.py:85]: this is breaking the basic existing functionality (connecting to orthanc etc..)
We can't have multiple prefix for the same kind..
/dicom
/dicom/wado etc..

**Pull request merged**